### PR TITLE
DENG-2988 - rename 4 columns in schema and query

### PR DESCRIPTION
**Pull request by kwindau**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-3025)

### CODE REVIEW

**Review by Marlene-M-Hirose — COMMENTED** [review]
I'm wondering why the column names need to be changed. @irrationalagent and @SuYoungHong were involved when I created this table. Leif and Su, do you have opinions?

Also, these columns were in line with the original cohort_daily_statistics table and renaming columns will cascade a need for many updates in Looker both the explore and dashboards

**Comment by SuYoungHong** [comment]
hey @Marlene-M-Hirose , I think this is related to @bani 's work on updating the definition of DAU. 
[DAU Follow Up ETL Tasks](https://docs.google.com/document/d/13AIlzxJafKJc0xBps9j4fyqH2h79Px0rr2uEvZwfabc/edit#heading=h.vlcnfd28ugmc)

File: sql/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/view.sql [hunk 1]
```diff
@@ -55,6 +55,6 @@ SELECT
   os_version_major,
   os_version_minor,
   num_clients_in_cohort,
-  num_clients_active_on_day
+  num_clients_sent_ping_on_day AS num_clients_active_on_day
```
Comment (bani) [inline sql/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/view.sql:58]: the goal is to also have it renamed on the views

**Comment by irrationalagent** [comment]
> I'm wondering why the column names need to be changed. @irrationalagent and @SuYoungHong were involved when I created this table. Leif and Su, do you have opinions?
> 
> Also, these columns were in line with the original cohort_daily_statistics table and renaming columns will cascade a need for many updates in Looker both the explore and dashboards

I think this is part of work to standardize what we call DAU-related measurements. @bani documented this work [here](https://docs.google.com/document/d/13AIlzxJafKJc0xBps9j4fyqH2h79Px0rr2uEvZwfabc/edit#heading=h.vlcnfd28ugmc)

**Pull request closed without merging**